tests/ansible_collections/roles/update-system/tasks/update-system.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Update system
+  yum:
+    name: "*"
+    state: latest
+  register: update
+
+- name: Reboot
+  reboot:
+  when: (ansible_facts['distribution'] == "CentOS") and
+        (update.msg != "Nothing to do")
```

Comment:
Shouldn't it be:
```suggestion
        ("Nothing to do" not in update.msg)
```

Reply:
I checked the message is exactly `Nothing to do`, thus I added this type of check

Reply:
What I'm getting with yum update is:
```
# yum update
Last metadata expiration check: 2:58:02 ago on Wed 19 May 2021 01:10:38 PM UTC.
Dependencies resolved.
Nothing to do.
Complete!
```
But the other messages maybe appear just in terminal, I don't know.

Reply:
I relied on centos8 tests, where:
```
            out: 
            out: TASK [update-system : Update system] *******************************************
            out: ok: [c2r_centos8_template-clone] => {"changed": false, "msg": "Nothing to do", "rc": 0, "results": []}
            out: 
```

Reply:
But you're right - now I am testing it on centos 7 and there is a different output

Reply:
Found a cleaner way to do this:
```
  when: (ansible_facts['distribution'] == "CentOS") and
        (update.changed)
```